octopus/tests/test_prf.py

```diff
@@ -0,0 +1,40 @@
+import os
+import numpy as np
+from astropy.io import fits
+
+from . import PATH_TESTDATA
+from .. import models
+from .. import core
+from ..kepler_prf import KeplerPRF
+
+
+def test_prf_normalization():
+    """Does the PRF model integrate to the requested flux across the focal plane?"""
+    for channel in [1, 20, 40, 60, 84]:
+        for col in [123, 678]:
+            for row in [234, 789]:
+                shape = (18, 14)
+                flux = 100
+                prf = KeplerPRF(channel=channel, column=col, row=row, shape=shape)
+                prf_sum = prf.evaluate(flux, col + shape[0]/2, row + shape[1]/2, 0).sum()
+                assert np.isclose(prf_sum, flux, rtol=0.1)
+
+
+def test_prf_vs_aperture_photometry():
+    """Is the PRF photometry result consistent with simple aperture photometry?"""
+    tpf = fits.open(os.path.join(PATH_TESTDATA, "ktwo201907706-c01-first-cadence.fits.gz"))
+    col, row = 173, 526
+    prf = KeplerPRF(channel=tpf[0].header['CHANNEL'],
+                    column=col, row=row,
+                    shape=tpf[1].data.shape)
+    fluxo, colo, rowo, bkgo = models.get_initial_guesses(data=tpf[1].data,
```

Comment:
``get_initial_guesses`` doesn't return a initial guess for the background =/
the last argument is ``sigma`` in case of Gaussian PSF

Reply:
I manually added a good `bkgo` estimate, but it still fails!